packages/suite/src/views/settings/general/Tor.tsx

```diff
@@ -1,27 +1,35 @@
 import React from 'react';
-import { Switch } from '@trezor/components';
+import { LoadingContent, Switch } from '@trezor/components';
 import { TOR_PROJECT_URL } from '@trezor/urls';
 import { useSelector, useActions } from '@suite-hooks';
 import { ActionColumn, SectionItem, TextColumn } from '@suite-components/Settings';
 import * as suiteActions from '@suite-actions/suiteActions';
 import { Translation } from '@suite-components';
 import { useAnchor } from '@suite-hooks/useAnchor';
 import { SettingsAnchor } from '@suite-constants/anchors';
-import { getIsTorEnabled } from '@suite-utils/tor';
+import { getIsTorEnabled, getIsTorLoading } from '@suite-utils/tor';
+import { TorStatus } from '@suite-reducers/suiteReducer';
```

Comment:
TorStatus is not defined in suiteReducer, I will send a fixup

Reply:
1ea3ff9e00e5667700011de59100c6a5ec03b530

Reply:
I'm too late :smiling_face_with_tear:

Reply:
Second place with the same bug: https://github.com/trezor/trezor-suite/pull/5666/commits/84a9ef40ece90e6cfeae87b431e3d88f6e2de61f